inc/admin/class-estore-theme-review-notice.php

```diff
@@ -0,0 +1,190 @@
+<?php
+/**
+ * eStore Theme Review Notice Class.
+ *
+ * @author  ThemeGrill
+ * @package eStore
+ * @since   1.4.6
+ */
+
+// Exit if accessed directly.
+defined( 'ABSPATH' ) || exit;
+
+/**
+ * Class to display the theme review notice for this theme after certain period.
+ *
+ * Class eStore_Theme_Review_Notice
+ */
+class eStore_Theme_Review_Notice {
+
+	/**
+	 * Constructor function to include the required functionality for the class.
+	 *
+	 * eStore_Theme_Review_Notice constructor.
+	 */
+	public function __construct() {
+
+		add_action( 'after_setup_theme', array( $this, 'estore_theme_rating_notice' ) );
+		add_action( 'switch_theme', array( $this, 'estore_theme_rating_notice_data_remove' ) );
+		add_action( 'admin_enqueue_scripts', array( $this, 'estore_theme_review_notice_enqueue' ) );
+
+	}
+
+	/**
+	 * Set the required option value as needed for theme review notice.
+	 */
+	public function estore_theme_rating_notice() {
+
+		// Set the installed time in `estore_theme_installed_time` option table.
+		$option = get_option( 'estore_theme_installed_time' );
+
+
+		if ( ! $option ) {
+			update_option( 'estore_theme_installed_time', time() );
+		}
+
+		add_action( 'admin_notices', array( $this, 'estore_theme_review_notice' ), 0 );
+		add_action( 'admin_init', array( $this, 'estore_ignore_theme_review_notice' ), 0 );
+		add_action( 'admin_init', array( $this, 'estore_ignore_theme_review_notice_partially' ), 0 );
+
+	}
+
+	/**
+	 * Display the theme review notice.
+	 */
+	public function estore_theme_review_notice() {
+
+		global $current_user;
```

Comment:
No need to define global variable, use WP functions like get_current_user_id(), wp_get_current_user() directly